wiki/en/Running-a-Server.md

```diff
@@ -50,55 +49,47 @@ Unless you plan on hosting more than about 5 players on a slower-speed home conn
-<figcaption>How Public Servers work</figcaption>
+	<img src="{% include img/en-screenshots/diagram-reg-server.inc %}" loading="lazy" alt="Diagram of connections between clients within a Jamulus Registered Server">
+<figcaption>How Registered Servers work</figcaption>
 </figure>
 
+### 2. Unregistered 
 
-
-### 2. Private
-
-This is the default when starting a server for the first time. Private Servers are not listed by Directories, so only musicians who know your server's address to will be able to connect to it. This is useful because Jamulus does not (in any server mode) let you control who can connect to a server.
-
+This is the default when starting a server for the first time. Unregistered Servers are not listed by Directories, so only musicians who know your server's address to will be able to connect to it. This is useful because Jamulus does not let you control who can connect to a server.
 
 <figure>
-	<img src="{% include img/en-screenshots/diagram-private-server.inc %}" loading="lazy" alt="Diagram of connections between clients within a Jamulus Private Server">
-	<figcaption>How Private Servers work</figcaption>
+	<img src="{% include img/en-screenshots/diagram-unreg-server.inc %}" loading="lazy" alt="Diagram of connections between clients within a Jamulus Unregistered Server">
+	<figcaption>How Unregistered Servers work</figcaption>
 </figure>
 
+### 3. Custom Directory
 
+If you want to run a number of Servers, possibly also behind a firewall or on a LAN, you may want to run your server as a Directory. Examples include online events, music associations, sectional rehearsals or music lessons for schools.
 
-### 3. Directory
-
-For when you want to run a number of Private Servers, possibly also behind a firewall or on a LAN. Examples include online events, music associations, sectional rehearsals or music lessons for schools.
-
-To view servers listed by a custom Directory Server, musicians must enter the address in their client’s "Custom Directory Server" settings field.  They will then see a Connection Setup list which is generated by your Directory.
+To view servers listed by a Custom Directory, musicians must enter its address in their client’s "Custom Directory Server" settings field.  They will then see a Connection Setup list which is generated by your Directory. Custom Directories otherwise work for clients in the same way as Public Directories.
 
 Up to 150 servers can then register with your Directory by setting your Directory address in their `--directoryserver` option at startup.
 
-To run a server as a Directory, it should be configured with `--directoryserver localhost` (that is, making itself the Directory to query for servers).
+To run a server as a Directory, it should be configured with `--directoryserver localhost` (that is, making itself the Directory to query for servers). When running a server with the GUI, select "Custom" from the Directory drop-down list. 
```

Comment:
> select "Custom" from the Directory drop-down list

... and enter "localhost" as the Custom Directory server address.

Reply:
Is there a reason why it doesn't do that automatically?

Reply:
The _vast_ majority of people registering with a Custom Directory are doing so as Servers.

They're _not_ setting up a Directory.

Otherwise you'd end up with more Directories than Servers... which makes no sense at all.

Reply:
Sorry, I meant when they choose the "Directory" option in the Directory drop-down. Or have I misunderstood what that means? So right now if I want to set my server up as a Directory, I have to declare that in two places in the UI?

Reply:
There is no "Directory" option in the Server "Directory" drop down.  You pick "Custom" and enter one of the two literal strings in the Custom Directory server address that you'd pass to the `-e` option on the command line.

Basically, "Custom" works the same way as the command line option.  ("None" works like not giving the `-e` on the command line; the genre values aren't available from the command line.)

Reply:
It might be simpler to strip _all_ mention of running as a Directory from the documentation except for the definition list and from there have a link to a single page that explains how to set one up both from the command line (the normal thing you'd do) and from the GUI.

Almost no one wants to know.

Reply:
Yes. I think so too. Probably we'd also stress the unregistered server part in the docs.
Setting up a registered (public) server is probably no longer that useful. We have a bunch of free ones all over the world (which are probably all running via CLI by more experienced users).

Reply:
Good idea. It's too confusing trying to keep it all together as different "states" of the server.